hack/hive/hive-dev-install.sh

```diff
@@ -40,16 +39,23 @@ main() {
 	fi
 	verify_tools
 
-	if [ "$( $KUBECTL get namespace $HIVE_OPERATOR_NS -o yaml 2>/dev/null | wc -l )" -ne 0 ]; then
-		log "hive is already installed in namespace $HIVE_OPERATOR_NS"
-			log -n "would you like to reapply the configs? (y/N): "
-			read answer
-			if [[ "$answer" != "y" ]]; then
-				exit
-		    fi
-	else
-		$KUBECTL create namespace $HIVE_OPERATOR_NS
-	fi
+    if [ "$( $KUBECTL get namespace $HIVE_OPERATOR_NS -o yaml 2>/dev/null | wc -l )" -ne 0 ]; then
```

Comment:
There's no need to get the yaml for the namespace, since the output is sent to `/dev/null`.